app/src/layouts/DashboardLayout/MenuHeader/PremiumPlanBadge/PremiumPlanBadge.tsx

```diff
@@ -76,6 +77,13 @@ const PremiumPlanBadge = () => {
     planId &&
     [APP_CONSTANTS.SUBSCRIPTION_STATUS.TRIALING, APP_CONSTANTS.SUBSCRIPTION_STATUS.CANCELLED].includes(planStatus)
   ) {
+    if (daysLeft > 30) {
+      return null;
+    }
+
+    const planName =
+      userPlanDetails?.planName === "professional" ? "Pro" : getPrettyPlanName(userPlanDetails?.planName);
```

Comment:
Lets handle `professional` plan in `getPrettyPlanName` itself

Reply:
Its handled there but this is a specific case where we want to show `Pro` instead of `Professional`

Reply:
Ok